wherehows-dao/src/main/java/wherehows/models/DatasetColumn.java

```diff
@@ -13,6 +13,10 @@
  */
 package wherehows.models;
 
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter @Setter
```

Comment:
Let's use @Builder instead of @Getter & @Setter: https://projectlombok.org/features/Builder